README.md

```diff
@@ -2,6 +2,7 @@ This repository powers multiple language support for
 [CodeCrafters](https://codecrafters.io) challenges.
 
 <!-- toc -->
+## Table of Contents
```

Comment:
@awong503 thanks for this! Minor change: Could you place this on top of the `<!-- toc -->` marker please? The table of contents is generated automatically using [this command](https://github.com/codecrafters-io/languages/blob/d883305d2532325a57696440045efd496de6a488/Makefile#L13-L14), and it'll replace anything inside the `<!-- toc -->` markers.

Reply:
Yup, will do! Thanks for pointing that out.